docs/dev/development/development-database.rst

```diff
@@ -8,25 +8,25 @@ This dump is created from the database behind test.pypi.org, not production.
 
 Use ``pg_dump`` to create the initial file::
 
-    $ pg_dump --no-owner $DATABASE_URL > dev/example.sql
+    pg_dump --no-owner $DATABASE_URL > dev/example.sql
 
 Now create a database locally and load the result::
 
-    $ createdb warehouse_dev_dump
-    $ psql warehouse_dev_dump < dev/example.sql
+    createdb warehouse_dev_dump
+    psql warehouse_dev_dump < dev/example.sql
 
 With the database loaded, run the clean script to remove personal information::
 
-    $ psql warehouse_dev_dump < dev/clean.sql
+    psql warehouse_dev_dump < dev/clean.sql
 
 Dump the result, compress it, and clean up::
 
-    $ pg_dump --no-owner warehouse_dev_dump > dev/example.sql
-    $ xz -z -9 dev/example.sql
-    $ dropdb warehouse_dev_dump
+    pg_dump --no-owner warehouse_dev_dump > dev/example.sql
+    xz -z -9 dev/example.sql
+    dropdb warehouse_dev_dump
 
 Now commit the result::
 
-    $ git checkout -b update_dev_db_dump
-    $ git add dev/example.sql.xz
-    $ git commit -m "Update development database dump"
+    git checkout -b update_dev_db_dump
+    git add dev/example.sql.xz
+    git commit -m "Update development database dump"
```

Comment:
These multi-line command blocks don't really copy correctly, we might want to break them into separate blocks instead.

Reply:
Can you explain what you mean? I tested the copy button, and my cliboard contents have:

```
git checkout -b update_dev_db_dump
git add dev/example.sql.xz
git commit -m "Update development database dump"
```

Reply:
Since copy/pasting this into a terminal would result in the first two lines immediately being executed due to the newlines, I would expect this to either be:

```
git checkout -b update_dev_db_dump
```
```
git add dev/example.sql.xz
```
```
git commit -m "Update development database dump"
```

or:

```
git checkout -b update_dev_db_dump; \
git add dev/example.sql.xz; \
git commit -m "Update development database dump"
```

Reply:
(fwiw, I think I prefer the former)

Reply:
I've set the continuation on these multi-line commands. FWIW, my terminal application (iTerm2) warns me whenever I paste in multi-line command input before it inputs them. I realize not everyone uses my terminal nor my settings.